Directory.Build.props

```diff
@@ -404,6 +404,13 @@
   <!-- Import it at the end of the props file to override the OutputPath for reference assemblies and use common directory props -->
   <Import Project="$(MSBuildThisFileDirectory)eng/ReferenceAssemblies.props" />
 
+  <PropertyGroup>
+    <!-- Set the documentation output file globally. -->
+    <DocumentationFile Condition="'$(IsTestProject)' != 'true' AND '$(IsReferenceAssembly)' !='true'">$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
```

Comment:
You can move this condition to the property group itself and then make DocumentationFile overridable. i.e:
```xml
<PropertyGroup Condition="condition">
  <DocumentationFile Condition="'$(DocumentationFile)' == ''">...<.../>
</PropertyGroup>
```

With that we wouldn't need the condition in the NoWarn property anymore :smile:

Reply:
Alright.